webpack.config.js

```diff
@@ -30,11 +34,55 @@ module.exports = {
       }]
     },
     {
-      test: /\.(png|bin)$/,
+      test: /\.png$/,
       loader: "url-loader",
       options: {
-        limit: true, // everything
+        limit: 8192,
+        outputPath: 'assets/images',
+        publicPath: 'images',
+        postTransformPublicPath: (p) => `__ml_activities_asset_public_path__ + ${p}`,
+        name: '[name].[ext]?[contenthash]',
+      }
+    },
+    {
+      type: 'javascript/auto',
+      test: /src\/demo\/model.json$/,
+      use: [
+          {
+            loader: 'file-loader',
+            options: {
+              outputPath: 'assets/models',
+              publicPath: 'models',
+              postTransformPublicPath: (p) => `__ml_activities_asset_public_path__ + ${p}`,
+              name: '[name].[ext]?[contenthash]',
+            }
+          }
+      ]
+    },
+    {
+      test: /\.(mp3|ogg|wav)$/,
+      loader: "file-loader",
+      options: {
+        outputPath: 'assets/sounds',
+        publicPath: 'sounds',
+        postTransformPublicPath: (p) => `__ml_activities_asset_public_path__ + ${p}`,
+        name: '[name].[ext]?[contenthash]',
       }
     }],
   },
+  performance: {
+    assetFilter: function (assetFilename) {
+      return (/^(images|models|sounds)/.test(assetFilename));
+    },
+    maxAssetSize: 300000,
+    maxEntrypointSize: 10500000,
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    new CopyPlugin([{
+      from: 'src/demo/*.bin',
+      to: 'assets/models/',
+      flatten: true,
+    }]),
```

Comment:
This handles copying the `.bin` file for the mobilenet model. Since we don't import the `.bin` file (it is just referenced from within the `model.json`, we need to explicitly copy it)